### ISSUE TITLE
grants - rounds 8 scope (aditya)

### ISSUE THREAD

**Issue by thelostone-mc** [gitcoinco/web#7714]
#### Feature

- Cleanup sidebar. Grant Types != Grant CLR.
- Show active CLRs on landing page and have them redirect to /grants/clr/<clr-name>  ?
- Re-write grant detail page in vue and remove inline edit and replace with modal (same code as grants/new)
- Update CLR to include trust verification scores  

#### Nice to have

- Ability to have collection specific to a GrantCLR   
- ~Ability to prefill fields based on URL during grant creation` /grant/new?type=matic`~

#### Tech-Debt

- Ensure mails are sent out only after txn are successful validated
- Lay-out new table for clr_prediction_curve (allows grants to be part of multiple CLR rounds at the same time) 
- ~Plug and Play checkout experience. Lay the foundation + documentation~

**Comment by owocki** [comment]
> Re-write grant detail page in vue and remove inline edit and replace with modal (same code as grants/new)

this might be good timing if we redesign the grants page this round too @willsputra @PixelantDesign !

**Comment by PixelantDesign** [comment]
New design is on the list. Will share on the call tomorrow.

**Comment by owocki** [comment]
TODO @thelostone-mc - break up into individual cards + put on grants round 8 project https://github.com/gitcoinco/web/projects/52
